tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -590,6 +601,32 @@ public:
    std::string GetActionName() { return "Take"; }
 };
 
+// Extern templates for TakeHelper
+// NOTE: The move-constructor of specializations declared as extern templates
+// must be defined out of line, otherwise cling fails to find its symbol.
+template <typename RealT_t, typename T, typename COLL>
+TakeHelper<RealT_t, T, COLL>::TakeHelper(TakeHelper<RealT_t, T, COLL> &&) = default;
+template <typename RealT_t, typename T>
+TakeHelper<RealT_t, T, std::vector<T>>::TakeHelper(TakeHelper<RealT_t, T, std::vector<T>> &&) = default;
+template <typename RealT_t, typename COLL>
+TakeHelper<RealT_t, RVec<RealT_t>, COLL>::TakeHelper(TakeHelper<RealT_t, RVec<RealT_t>, COLL> &&) = default;
+template <typename RealT_t>
+TakeHelper<RealT_t, RVec<RealT_t>, std::vector<RealT_t>>::TakeHelper(TakeHelper<RealT_t, RVec<RealT_t>, std::vector<RealT_t>> &&) = default;
+
+// External templates are disabled for gcc5 since this version wrongly omits the C++11 ABI attribute
+#if !(__GNUC__ <= 5)
```

Comment:
if there is a tricky reason why `!(__GNUC__ <= 5)` is better than `__GNUC__ > 5` it should be commented

Reply:
No reason, only own strange logic :D